packages/weblets/src/elements/caprover/Caprover.wc.svelte

```diff
@@ -312,7 +312,7 @@
       5- Click <strong>Join cluster</strong> button<br />
       <br />
       <strong>
-        <a target="_blank" href="https://library.threefold.me/info/manual/#/manual__weblets_caprover_worker">
+        <a target="_blank" href="https://manual.grid.tf/weblets/weblets_caprover.html">
```

Comment:
I think this should be `https://manual.grid.tf/weblets/weblets_caprover_worker.html`